### PR TITLE
Minor content changes

### PR DESCRIPTION
**Pull request by vibhabamba**
Updated resume with links
Updated resume file
Added 'compose email hook' from main page

### CODE REVIEW

**Comment by zahanm** [comment]
> Added 'compose email hook' from main page

Lol, what is the idea here?
That just opens Gmail. Hah, I don't think you want this at all.

**Comment by vibhabamba** [comment]
The idea is for it to open up a compose window. Am i doing something wrong?

On Tuesday, June 30, 2015, Zahan Malkani notifications@github.com wrote:

> Added 'compose email hook' from main page
> 
> Lol, what is the idea here?
> That just opens Gmail. Hah, I don't think you want this at all.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/zahanm/vibhabamba.com/pull/18#issuecomment-117342630.

## 

Vibha Bamba

**Comment by zahanm** [comment]
https://css-tricks.com/snippets/html/mailto-links/

**Comment by vibhabamba** [comment]
ok fixed now.

**Comment by vibhabamba** [comment]
Also fixed broken resume link and email icon.

**Comment by vibhabamba** [comment]
Ping

File: index.html [hunk 1]
```diff
@@ -41,22 +41,26 @@
...
         <img src="/images/resume.png" />
-        <a href="VibhaBamba-June-2015.pdf">
-          resume
+        <a href="VibhaBamba_2015.pdf" target="_blank">
+          Resume
         </a>
       </li>
+      <li>
+        <img src="/images/email.png" />
+        <a href="mailto:vibhabamba11@gmail.com">Contact</a>
```
Comment (zahanm) [inline index.html:62]: you sure you want this publicly available? okay.

**Comment by zahanm** [comment]
Screenshot? This is good to go.

File: index.html (hunk 1 above)
Comment (vibhabamba) [inline index.html:62]: yeah I think its fine.

**Pull request merged**